modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -6,192 +6,236 @@
-sudo apt-get install libssl-dev
+sudo apt install libssl-dev
 
-python3 -m  pip install couchbase
+sudo python3 -m pip install couchbase
 
 ----
 
 .RHEL and CentOS
-[source,console]
+[source,bash]
 ----
 # Only needed during first-time setup:
-sudo yum install gcc gcc-c++ python-devel python-pip cmake
+sudo yum install gcc gcc-c++ python3-devel python3-pip cmake
 
 # for TLS/SSL support (optional)
 sudo yum install openssl-devel
 
-python -m pip install couchbase
+sudo python3 -m pip install couchbase
 ----
 
-NOTE: RHEL/CentOS distributions may not provide the `python-pip` package in the base repositories.
+NOTE: RHEL/CentOS distributions may not provide the `python3-pip` package in the base repositories.
 It may be found in the https://fedoraproject.org/wiki/EPEL[EPEL^] repository.
 
+NOTE: Cmake may also need to be updated.  IF so, follow the steps https://idroot.us/install-cmake-centos-8[HERE^].
+
 == Installation on Mac OS X
 
-To install the library on Mac OS X, first install the de-facto package manager for OS X: http://brew.sh/[Homebrew^].
-Once _Homebrew_ is configured:
+The following example uses the Python supplied by the Homebrew package manager and not the vendor-supplied Python which ships with OS X.  
 
-[source,console]
-----
-$ brew update # get list of latest packages
-----
+To install the library on Mac OS X, first install Homebrew: http://brew.sh/[Homebrew^]. Once _Homebrew_ is configured:
 
-[source,console]
-----
-$ brew install python
-----
+NOTE: Later versions of Mac OS X break the python3 homebrew installer.  Simple mitigating steps may be found https://stackoverflow.com/questions/47255517/brew-install-python3-didnt-install-pip3[HERE^].
 
-[source,console]
-----
-$ brew install openssl # for TLS/SSL support - optional
-----
 
-[source,console]
+[source,bash]
 ----
-$ python -m pip install couchbase
+brew update # get list of latest packages
+brew install python3
+brew install openssl # for TLS/SSL support - optional
+
+python3 -m pip install couchbase
 ----
 
-NOTE: The above example uses the Python supplied by Homebrew and not the vendor-supplied Python which ships with OS X.
-The Python SDK will still work with the vendor-supplied Python (though _pip install_ may be a privileged command), but it is recommended to use Homebrew's Python instead.
+
 
 == Installing on Microsoft Windows
 
-[source,console]
+[source,shell]
 ----
-$ python -m pip install couchbase
+ppython3 -m pip install couchbase
 ----
 
-The standard Python distributions for Windows include OpenSSL DLLs, as PIP and the inbuilt `ssl` module require it for correct operation. 
-The binary wheels for Windows are packaged as a binary wheel built against the relevant version OpenSSL (which is fixed per Windows version of Python).
-
-If you require a version without OpenSSL support, or that doesn't have a suitable binary wheel on PyPi, follow the build instructions on the GitHub repo.
+NOTE: Because the Windows client is packaged as binary wheel,
```

Comment:
Revert, this must be based on an older set of instructions.

Reply:
please make this change as you see fit.  I have no opinion on this part.